### ISSUE TITLE
Allow navigation from the "this crate has been yanked" page to the normal crate info page

### ISSUE THREAD

**Issue by sfackler** [rust-lang/crates.io#502]
If you're trying to track down e.g. the author of a yanked crate, it's currently pretty hard to do.

**Comment by shepmaster** [comment]
You mean something other than navigation to the list of all versions, yes? For example, when I see a yanked page, it has a link to the other versions:

![image](https://cloud.githubusercontent.com/assets/174509/22159460/00bc5972-df0f-11e6-9e85-9e1cb09c0cbb.png)

Are you saying you'd like a way to see the content that *would* have appeared on `https://crates.io/crates/foo/1.2.3`, were the crate not yanked?

**Comment by mathstuf** [comment]
Yes, that way authors, repositories, documentation, etc. can still be accessed.

**Comment by dpc** [comment]
I'd like to suggest that if latest version have been yanked, the latest not-yanked version should be used. 

Eg. https://crates.io/crates/slog-syslog should just show version 0.8.2, like it's business as usual.

**Comment by shepmaster** [comment]
@dpc you mean #510 ?

**Comment by dpc** [comment]
@shepmaster Yes. :)

**Comment by carols10cents** [comment]
Yes, @dpc, this issue is for the case when *all* versions of a crate have been yanked. Sorry for the confusion!
